Micr.cs

```diff
@@ -124,7 +130,7 @@ public static bool IsValid( string content )
                     return false;
                 }
 
-                if ( micr.GetAccountNumber().Length < 6 )
```

Comment:
Checking the GetAccountNumber was a required thing for us as it caught a number of improper MICR reads from the check scanner. Does Wells Fargo use account numbers less than 6 digits?